web/src/infra/common/components/game/GameCard.module.css

```diff
@@ -0,0 +1,68 @@
+.Main {
+  position: relative;
+  height: 250px;
+  width: 100%;
+  background-position: left center;
+  background-size: cover;
+  color: black;
+}
+
+.Main--link {
+  box-shadow: 0 3px 6px rgba(0, 0, 0, 0.16), 0 3px 6px rgba(0, 0, 0, 0.23);
+  border-radius: 8px;
+}
+
+.Floating.Heading__Floating {
+  display: flex;
+  flex-wrap: wrap;
+  grid-gap: 8px;
+  top: 12px;
+  left: 8px;
+}
+
+.Panel.Title__Panel {
+  padding: 0px 8px;
+  padding-top: 4px;
+}
+
+.Warning__Icon.Warning__Icon {
+  width: 32px;
+  height: 100%;
+  padding: 0 4px;
```

Comment:
What do you think about adding `min-height: 45px` here? This way, the button will have the same height interdependently if it shows on the first line or the second line.

Reply:
Actually, 42px